templates/tfengine/components/cicd/README.md

```diff
@@ -1,21 +1,93 @@
-# Continuous Integration (CI) and Continuous Deployment (CD) component
+# Continuous integration (CI) and continuous deployment (CD)
 
-This directory contains the Terraform configs for the Continuous Integration (CI)
-and Continuous Deployment (CD) component.
+This directory defines the resources that need to be deployed manually to set up
+CICD pipelines of Terraform configs.
 
-* The [manual/](./manual) directory contains the Terraform resources that need to
-  be deployed manually to provision the initial CI/CD resources.
+The CI and CD pipelines use
+[Google Cloud Build](https://cloud.google.com/cloud-build) and
+[Cloud Build triggers](https://cloud.google.com/cloud-build/docs/automating-builds/create-manage-triggers)
+to detect changes in the repo, trigger builds, and run the workloads.
 
-* The [auto/](./auto) directory contains the Terraform resouces for CI/CD purposes
-  that are also managed by CI/CD itself.
+1. Generate the CICD Terraform configs and Cloud Build configs using the
+    Terraform Engine. Only the Terraform resources in the current directory,
+    that is, the `cicd/` directory under root, needs to be deplopyed manually.
 
-  Sensitive resources in the devops project such as
+1. Before deploying CICD Terraform resources, install the Cloud Build app and
+    connect your GitHub repository to your Cloud project by following the steps
+    in
+    [Installing the Cloud Build app](https://cloud.google.com/cloud-build/docs/automating-builds/create-github-app-triggers#installing_the_cloud_build_app).
+    To perform this operation, you need Admin permission in that GitHub
+    repository. This can't be done through automation.
 
-  * Terraform state bucket
-  * IAM bindings
-  * Cloud Build Triggers
+1. After the GitHub repo is connected, run the following commands in this
+    directory to enable the necessary APIs, grant the Cloud Build service
+    account the necessary permissions, and create Cloud Build triggers:
 
-   should not be put here to let CI/CD manage them, which could lead to
-   potential misconfiguration of CI/CD itself. Those resources should be
-   included in [manual/main.tf](./manual/main.tf) and deployed manually by
-   a human owner of the devops project.
+    ```shell
+    terraform init
+    terraform plan
+    terraform apply
+    ```
+
+    Two presubmit triggers are created by default. Build/test status and results
```

Comment:
I think this should mention that while results are posted in the PR, detailed logs are not, and you may need permissions to view them.